test/api_projects_test.go

```diff
@@ -33,7 +33,9 @@ func createTempUser(t *testing.T, apiClient *client.APIClient, prefix string) *c
 
 	resp, httpRes, err := apiClient.UsersAPI.CreateUser(context.Background()).CreateUserSchema(createUserSchema).Execute()
 
-	fmt.Println(err)
+	if err != nil {
```

Comment:
Small detail to make this silent when the test is green, just makes the output easier to read